fighthealthinsurance/common_view_logic.py

```
@@ -1562,33 +1602,52 @@ async def generate_appeals(cls, parameters) -> AsyncIterator[str]:
             )
+            pubmed_context = results[0]
+            ml_citations_context = results[1]
         except Exception as e:
-            logger.opt(exception=True).debug(
-                f"Error {e} looking up context for {denial}."
-            )
-        logger.debug("Pubmed context done.")
+            logger.debug(f"Error gathering contexts: {e}")
+            # We still might have saved a context.
+            denial.refresh_from_db()
+            pubmed_context = debial.pubmed_context
```

Ruff (0.8.2)

`debial` may be undefined, or defined from star imports

(F405)